packages/ui-sales/src/deals/components/PortableChooser.tsx

```
@@ -0,0 +1,97 @@
+          !fetchMoreResult ||
+          fetchMoreResult?.[options.queriesName.itemsQuery]?.length === 0
+        ) {
+          return prevResult;
+        }
+
+        const prevItems = prevResult?.[options.queriesName.itemsQuery] || [];
+        const prevItemIds = prevItems.map((item) => item._id);
+
+        const fetchedItems: any[] = [];
+
+        for (const item of fetchMoreResult?.[options.queriesName.itemsQuery]) {
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)